cfgov/unprocessed/js/organisms/FilterableListControls.js

```diff
@@ -51,9 +51,9 @@ function FilterableListControls( element ) {
     },
     email: function( field, currentStatus ) {
       var status = currentStatus || {};
-      var regex = [ "^[a-z0-9\u007F-\uffff!#$%&'*+\/=?^_`{|}~-]+(?:\.[a-z0-9",
-                    "\u007F-\uffff!#$%&'*+\/=?^_`{|}~-]+)*@(?:[a-z0-9]",
-                    "(?:[a-z0-9-]*[a-z0-9])?\.)+[a-z]{2,}$" ].join( '' );
+      var regex = [ '^[a-z0-9\u007F-\uffff!#$%&\'*+\/=?^_`{|}~-]+(?:\.[a-z0-9',
+                    '\u007F-\uffff!#$%&\'*+\/=?^_`{|}~-]+)*@(?:[a-z0-9]',
+                    '(?:[a-z0-9-]*[a-z0-9])?\.)+[a-z]{2,}$' ].join( '' );
```

Comment:
Can these be strings that are concatenated together directly instead of an array that is converted into a string? Also, a regex literal might be an option that's delimited by a beginning and ending `/`. Though that may be broken when spanning multiple lines.